content/docs/common-areas/_index.md

```diff
@@ -29,11 +29,13 @@ APM or Application Performance Monitoring can be defined as a process or a pract
 
 ## Backups
 
-## CDN
+## [CDN](/docs/common-areas/cdn)
+CDN helps cache content at the network edge, which improves website performance. Many websites struggle to have their performance needs met by traditional hosting services, which is why they opt for CDNs.
```

Comment:
This is unrelated to this PR. If I merge just this one, it will result in a broken link.